modules/mean_variance_frontier.py

```diff
@@ -0,0 +1,56 @@
+
+# Function to calculate and plot the Mean Variance Frontier
+
+import random
+import numpy as np
+import pandas as pd
+import seaborn as sns
```

Comment:
Delete seaborn, as it is not used.  And also, add:

`from matplotlib.axes._axes import _log as matplotlib_axes_logger
matplotlib_axes_logger.setLevel('ERROR')`

which will get rid of error in app